compiler/Fortran/crayftn/unlimited_polymorphic_assumed_rank_arg/inverted.f90

```diff
@@ -0,0 +1,17 @@
+module a
+contains
+  subroutine foo(x)
+    class(*), intent(in) :: x(..)
+    select rank (x)
```

Comment:
@everythingfunctional have you tried combined.f90 and inverted.f90 with `nagfor`?  I'm not certain that both are standard-conforming.  Certainly at least one of the two is standard-conforming, but if it's just one, then I'm not sure which one.

Reply:
You're right. The `select rank` has to be done first, as assumed rank entities are not allowed in `select type`. I've made the change to account for that.